indico/htdocs/sass/modules/_roles.scss

```diff
@@ -15,55 +15,55 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
-
-.role-list-row .i-tag {
-    display: inline-flex;
-    justify-content: center;
-    align-items: baseline;
-}
-
-.i-tag:not(:first-child) {
-    margin-left: 0.2em;
-}
-
 .role-list-row {
-    background: #eee;
-}
-
-.role-list-row td {
-    padding: 0 !important;
-}
 
-.role-list-row td .content-area {
-    background: #fff;
-    margin: 0 0 0 58px !important;
-}
+    background: #eee;
 
-.role-list-row td .content-area .empty {
-    padding: 10px;
-    font-style: italic;
-    color: $dark-gray;
-}
-
-.role-list-row td .content-area .toolbar {
-    padding: 10px;
+    td {
+        padding: 0 !important;
+
+        .content-area {
+            background: #fff;
+            margin: 0 0 0 58px !important;
+
+            .empty {
+                padding: 10px;
+                font-style: italic;
+                color: $dark-gray;
+            }
+
+            .toolbar {
+                padding: 10px;
+            }
+        }
+    }
+
+    .i-tag {
+        display: inline-flex;
+        justify-content: center;
+        align-items: baseline;
+
+        :not(:first-child) {
```

Comment:
`&:not(..) {` - otherwise it's matching a child element with that selector and not the element itself